config/develop/glue-database.yaml

```diff
@@ -0,0 +1,6 @@
+template_path: glue-database.yaml
+stack_name: glue-database
+parameters:
+  DatabaseName: bridge-downstream-etl
+stack_tags:
+  {{ stack_group_config.default_stack_tags }}
```

Comment:
I anticipate that this will eventually not be a free-standing stack, but will rather be part of a nested stack created per-study.